retry/retry.go

```diff
@@ -197,5 +198,11 @@ func Retryable(err error) bool {
 		return true
 	}
 
+	// For errors without a five-digit code, github.com/lib/pq uses fmt.Errorf().
+	// That returns an unexported error type, so we can only check the error message:
```

Comment:
Please link the created pq issue and also reference it in the commit description.